app/_src/gateway/kong-enterprise/signing-verify/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Verify signatures for Signed Kong Images
+badge: enterprise
+---
+
+As of {{site.ee_product_name}} version 3.5, Docker container images are now signed using `cosign` with signatures published to a Docker Hub repository.
+
+This guide provides steps to verify signatures for signed {{site.ee_product_name}} Docker container images in 2 different ways. The first is the minimal example, used to verify an image without leveraging any annotations. The second is the complete example, leveraging optional annotations for increased trust.
+
+For the minimal example, you only need: Docker details, Github repo name, and Github workflow filename.
+For the complete example, you will need the same details from above, as well as any of the optional annotations you wish to verify:
+
+- Github commit sha
+- Github repository
+- Github workflow name
+
+Because Kong uses Github Actions to build and release, Kong also uses Github's OIDC identity to sign images; thus many of these details are Github-related.
+
+## Examples
+
+For both examples, you will need to:
+
+1. Ensure `cosign` is installed
+
+2. Collect the necessary image details
+
+3. Set the `COSIGN_REPOSITORY` environment variable (`kong/notary` or `kong/notary-internal`)
+
+   ```sh
+   export COSIGN_REPOSITORY=kong/notary-internal
```

Comment:
@curiositycasualty Since this is user-facing docs, can we have this to `kong/notary` in the example